src/main/java/org/threadly/litesockets/protocol/http/structures/HTTPAddress.java

```diff
@@ -1,5 +1,10 @@
-package org.threadly.litesockets.protocol.http;
+package org.threadly.litesockets.protocol.http.structures;
 
+/**
+ * Simple Immutable class that is used for http Connections.  Its mainly used
+ * to know if 2 address will connect to the exact same place or not. 
+ *
+ */
 public class HTTPAddress {
```

Comment:
Why not use an InetSocketAddress?  Only information that would be missing is the "doSSL" which seems unrelated to the address.